### PR TITLE
form to add locations

### PR DESCRIPTION
**Pull request by aaaleeenaaa**
form to add new locations (fixed issues)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,6 +27,7 @@ yarn-error.log*
 
 # local env files
 .env*.local
```
Comment (t0bbl) [inline .gitignore:29]: i think .env*.local includes .env but maybe some one can confirm

File: components/AddButton.js
Comment (t0bbl) [inline components/AddButton.js:1]: you could use a "StyledButton.js" for just the styling and maybe a component for Button that is universal and edited with props.

File: components/Card.js [hunk 2]
```diff
@@ -17,11 +17,13 @@ const StyledListElement = styled.li`
   list-style-type: none;
 `;
 
-export default function Card() {
+export default function Card({ location }) {
   return (
     <StyledListElement>
       <StyledLink href="#">
-        <CardElement>Name of Location</CardElement>
+        <CardElement h>
```
Comment (t0bbl) [inline components/Card.js]: is the "h" a prop from styled components or a typo?

File: components/Form.js
Comment (t0bbl) [inline components/Form.js:1]: if you had a StyledButton component you could just import these here and use a prop like 
`${(props) =>
    props.submit &&
    css`
        padding: 0.5em;
        width: 20%;
    `}`

**Review by t0bbl — COMMENTED** [review]
LGTM, the comments i did are just for readability, maybe someone else can look over this.

File: .gitignore (hunk 1 above)
Comment (toshydev) [reply .gitignore:29]: No it doesn't include `.env`.
It's better to include it but in any case, working with `.env.local` should be sufficient during development. `.env` keys and values have to be manually uploaded in the server provider,  e.g. _Vercel_.

File: db/connect.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import mongoose from "mongoose";
+
+const MONGODB_URI = process.env.MONGODB_URI;
+if (!MONGODB_URI) {
+  console.log(MONGODB_URI, "WTF COUNT?");
```
Comment (toshydev) [inline db/connect.js]: You should remove unnecessary logging.
suggestion```

```

**Review by toshydev — APPROVED** [review]
LGTM ✅️
_Hint_: See comment.

File: db/connect.js (hunk 3 above)
Comment (aaaleeenaaa) [reply db/connect.js]: Thanks, removed it!

File: components/Card.js (hunk 2 above)
Comment (aaaleeenaaa) [reply components/Card.js]: Just a typo, removed it, thanks!

File: .gitignore (hunk 1 above)
Comment (aaaleeenaaa) [reply .gitignore:29]: Just came up with that while trying around, deleted it again for the moment as for now it's not yet relevant. But thanks for the insights, probably useful for later.

File: components/AddButton.js
Comment (aaaleeenaaa) [reply components/AddButton.js:1]: That's a good point, thanks! As, however, the add and submit button I currently have don't have anything in common (well, except from both being buttons), I would leave it like that for now. But I will definitely keep that in mind and have a look at it again later on.

**Pull request merged**